packages/redux/src/Oidc.spec.js

```diff
@@ -1,16 +1,40 @@
 // Link.react.test.js
 import React from 'react';
 import renderer from 'react-test-renderer';
-import { OidcBase } from './Oidc';
+import { shallow } from 'enzyme';
+import { loadUser } from 'redux-oidc';
 
-describe('redux-fetch.withAuthentication', () => {
+import Oidc, { OidcBase } from './Oidc';
+
+// I used __esModule to mock a module with default and named exports
+// cf: https://github.com/facebook/jest/issues/5579#issuecomment-397406174
+jest.mock('./authenticationService', () => ({
+  __esModule: true,
+  default: jest.fn(),
+  getUserManager: jest.fn(),
+}));
+jest.mock('redux-oidc', () => ({
+  OidcProvider: jest.fn(),
```

Comment:
Thank you very much Julien, very nice PR.

Do you know this article below ?
https://medium.com/javascript-scene/mocking-is-a-code-smell-944a70c90a6a

I think it is possible to remove the mock and make your code more flexible and testable :)
Your welcome at my desk for some ideas !

Reply:
I made some changes. Using a class component seemed to be a problem, so I used defaultProps enhancer from recompose. I still had to mock loadUser to check if it is called when the component mounts.